CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ Details:
 - Purpose: official release
 - Suitable environment: production
 
+## New Features
```

Comment:
Can you remove any change to the changelog? changelog entry will be generated automatically from now on based on the pull request title

Reply:
I have removed the entry from CHANGELOG.md file